envoy/type/v2/range.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+// [#proto-status: draft]
+
+package envoy.type.v2;
```

Comment:
Please just use envoy.type. Any package that ends with `.type` promises never have any breaking change, like `google.type`. In some cases, we should bite the bullet to make hard promise.

This is one reason I encourage Envoy and Istio to take dependency on google.type and google.rpc. Google APIs promise not to make any breaking change to them.